include/mega/logging.h

```diff
@@ -292,7 +292,15 @@ class SimpleLogger
 #endif
     {
 #ifdef ENABLE_LOG_PERFORMANCE
-        logValue(filename);
+        const char * actualFileName = strrchr(filename, '/'); //The project part of the path uses `/` for Windows too.
+        if (actualFileName)
```

Comment:
Perhaps protect against filenames like: `mypath/to/some/folder/`:
```suggestion
        if (actualFileName && *actualFileName != '\0')
```